installer/installer.py

```diff
@@ -412,9 +423,13 @@ def install(self):
         self.prepare_directory_tree()
         self.setup_system_configs()
 
-        if self.interviewer.is_cosmo_needed:
+        if self.interviewer.is_cosmo_needed and self.interviewer.is_cosmovisor_bump_needed:
```

Comment:
This condition might not be needed since the interviewer handles whether cosmo needed/bump needed is required.